core-services/comment/src/main/java/org/silverpeas/core/comment/service/CommentUserNotification.java

```diff
@@ -23,46 +23,51 @@
  */
 package org.silverpeas.core.comment.service;
 
+import org.silverpeas.core.comment.model.Comment;
 import org.silverpeas.core.contribution.model.SilverpeasContent;
 import org.silverpeas.core.contribution.model.SilverpeasToolContent;
-import org.silverpeas.core.comment.model.Comment;
+import org.silverpeas.core.notification.user.FallbackToCoreTemplatePathBehavior;
 import org.silverpeas.core.notification.user.builder.AbstractTemplateUserNotificationBuilder;
-import org.silverpeas.core.notification.user.model.NotificationResourceData;
 import org.silverpeas.core.notification.user.client.constant.NotifAction;
-import org.silverpeas.core.util.LocalizationBundle;
-import org.silverpeas.core.util.SettingBundle;
+import org.silverpeas.core.notification.user.model.NotificationResourceData;
 import org.silverpeas.core.template.SilverpeasTemplate;
-import org.silverpeas.core.template.SilverpeasTemplateFactory;
+import org.silverpeas.core.util.LocalizationBundle;
 
 import java.util.Collection;
 import java.util.MissingResourceException;
-import java.util.Properties;
 import java.util.Set;
 
 import static org.silverpeas.core.util.StringUtil.isDefined;
 
 /**
  * @author Yohann Chastagnier
  */
-public class CommentUserNotification extends AbstractTemplateUserNotificationBuilder<SilverpeasContent> {
-
-  /**
-   * If no property with the subject of the notification message is defined in a Silverpeas component, then the below
-   * default property is taken.
-   */
-  public static final String DEFAULT_SUBJECT_COMMENT_ADDING = "comments.commentAddingSubject";
+public class CommentUserNotification
+    extends AbstractTemplateUserNotificationBuilder<SilverpeasContent>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "SilverpeasContent"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)